Xamarin.Forms.Platform.UAP/CompositionHelper.cs

```diff
@@ -14,8 +16,11 @@ public static bool IsCompositionGeometryTypePresent
 			{
 				if (!SetTypePresent)
 				{
-					MethodInfo methodInfo = typeof(Compositor).GetMethod("CreateEllipseGeometry");
-					IsTypePresent = methodInfo != null;
+					var compositorMethods = typeof(Compositor).GetMethods();
+					var createGeometricClipExists = compositorMethods.Any(m => m.Name == "CreateGeometricClip");
+					var createEllipseGeometryExists = compositorMethods.Any(m => m.Name == "CreateEllipseGeometry");
```

Comment:
this could have been done in a single iteration, but as it's done only once, I prefer the simplicity over the limited speed enhancement